src/frontend/sharedComponents/icons/Circle.js

```diff
@@ -2,28 +2,42 @@
 import * as React from "react";
 import { StyleSheet, View } from "react-native";
 import type { ViewStyleProp } from "../../types";
+import validateColor from "validate-color";
 
 type CircleProps = {
+  color?: string,
+  borderWidth?: number,
   radius?: number,
   children: React.Node,
   style?: ViewStyleProp,
 };
 
-const Circle = ({ radius = 25, style, children }: CircleProps) => (
-  <View
-    style={[
-      styles.circle,
-      {
-        width: radius * 2,
-        height: radius * 2,
-        borderRadius: radius * 2,
-      },
-      style,
-    ]}
-  >
-    {children}
-  </View>
-);
+const Circle = ({
+  color,
+  borderWidth = 2,
```

Comment:
Remove this prop since it's somewhat redundant with the `style` prop and because if a valid color is passed, safer to be opinionated and use a specific value defined within this component for the border width

Reply:
> safer to be opinionated and use a specific value defined within this component for the border width

ah this part doesn't hold for this use case because it's possible for a category to not have a color defined while the others do, and in that case we want a consistent border width.